### PR TITLE
Add DB issuer and verifiers for vc2.0.

### PR DESCRIPTION
**Pull request by JSAssassin**
I haven't tested this, so please test before merging. Thanks!

### CODE REVIEW

File: implementations/DigitalBazaar.json [hunk 1]
```diff
@@ -89,7 +96,25 @@
...
+    "endpoint": "https://verifier.qa.veres.app/verifiers/z19mFGL4n9MeFiNzPbge5gJrz/presentations/verify",
+    "capability": "{\"@context\":[\"https://w3id.org/zcap/v1\",\"https://w3id.org/security/suites/ed25519-2020/v1\"],\"id\":\"urn:uuid:452d121c-c69e-4338-9a03-cd20e8d2a85a\",\"controller\":\"did:key:z6MkptjaoxjyKQFSqf1dHXswP6EayYhPQBYzprVCPmGBHz9S\",\"parentCapability\":\"urn:zcap:root:https%3A%2F%2Fverifier.qa.veres.app%2Fverifiers%2Fz19mFGL4n9MeFiNzPbge5gJrz\",\"invocationTarget\":\"https://verifier.qa.veres.app/verifiers/z19mFGL4n9MeFiNzPbge5gJrz/presentations/verify\",\"expires\":\"2024-07-20T17:03:57Z\",\"proof\":{\"type\":\"Ed25519Signature2020\",\"created\":\"2023-07-21T17:03:58Z\",\"verificationMethod\":\"did:key:z6MkkkanLE2mcwABMS3RrN4S81E5zo3SSjo7Lc6EBKB4w5ck#z6MkkkanLE2mcwABMS3RrN4S81E5zo3SSjo7Lc6EBKB4w5ck\",\"proofPurpose\":\"capabilityDelegation\",\"capabilityChain\":[\"urn:zcap:root:https%3A%2F%2Fverifier.qa.veres.app%2Fverifiers%2Fz19mFGL4n9MeFiNzPbge5gJrz\"],\"proofValue\":\"z23LjDXNW2cdKh65JpG4krCU5R2X3D9NpRQQpXLGbrLKtmh9XEfeFGf6QgpxkR15Ui6UHjuD6dMfuU4GYAWL4jSPj\"}}",
+    "zcap": {
+      "keySeed": "KEY_SEED_DB"
+    },
+    "tags": ["vcdm2"]
```
Comment (aljones15) [inline implementations/DigitalBazaar.json:115]: this should be `vc2.0` also.

Also this should be in a new property `vpVerifiers`

**Review by aljones15 — CHANGES_REQUESTED** [review]
one small change

File: implementations/DigitalBazaar.json (hunk 1 above)
Comment (aljones15) [reply implementations/DigitalBazaar.json:115]: I made this change locally and am testing right now so don't worry about it.

Comment (JSAssassin) [reply implementations/DigitalBazaar.json]: Updated [d5a90ae](https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/72/commits/d5a90ae402c635c5623711d9be21a183f1c1963a)

**Review by aljones15 — APPROVED** [review]
LGTM

**Pull request merged**